packages/marshal/test/test-dot-membrane.js

```diff
@@ -1,6 +1,8 @@
 import { test } from './prepare-test-env-ava.js';
+
+// eslint-disable-next-line import/order
```

Comment:
why is this suppression necessary?

Reply:
Because of the side effects performed by the `./prepare-test-env-ava.js` import, it must come first. But we apparently have a lint rule that effectively says that `./...` imports must come after `@...` imports. When I remove this suppression and `yarn lint` I get

```
/Users/markmiller/src/ongithub/endojs/endo/packages/marshal/test/test-dot-membrane.js
  3:1  error  `@endo/pass-style` import should occur before import of `./prepare-test-env-ava.js`  import/order

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

In this case ;) vscode gives feedback that agrees with `yarn lint`

<img width="692" alt="image" src="https://user-images.githubusercontent.com/273868/212498737-f7bf6089-5b26-447c-8d8a-77e44a88b504.png">

Reply:
Even before this PR, we seem to have a lot of these in our system

<img width="582" alt="image" src="https://user-images.githubusercontent.com/273868/212498891-cc4b26e6-c24a-4243-b4c0-6cd0ad04f48d.png">

so it seems out of place to add an inline comment for this one in particular.

Reply:
See https://github.com/endojs/endo/issues/1446 . With that filed, resolving.